api/controllers/main.js

```diff
@@ -1,12 +1,19 @@
 const fs = require('fs');
 const path = require('path');
+const SiteWideErrorLoader = require('../services/SiteWideErrorLoader');
 
 const manifestPath = path.join(__dirname, '..', '..', 'webpack-manifest.json');
 const webpackAssets = JSON.parse(fs.readFileSync(manifestPath, 'utf-8'));
 
 module.exports = {
   index(req, res) {
+    let siteWideError = { display: false };
+    if (req.session.authenticated) {
+      siteWideError = SiteWideErrorLoader.loadSiteWideError();
+    }
+
     res.render('index.html', {
+      siteWideError,
```

Comment:
Yeah, the display prop make sense in the binary work of error / no error, but doesn't carry over into the world of looking for errors and authenticated sessions.